src/test/java/fi/helsinki/cs/tmc/core/commands/DownloadExercisesTest.java

```diff
@@ -118,7 +118,7 @@ public void writesChecksumsToCacheIfCacheFileIsGiven()
     public void downloadAllExercises() throws Exception {
         CoreTestSettings settings1 = createSettingsAndWiremock();
         core = new TmcCore(settings1);
-        Path folder = Paths.get(System.getProperty("user.dir") + "/testResources/");
+        Path folder = Paths.get(System.getProperty("user.dir") , "testResources");
```

Comment:
Is this doing what I think it is -- using a normal directory as temp directory - outside or temp folders...